app/server/prisma/seed.ts

```diff
@@ -3,9 +3,15 @@ import { PrismaClient } from '../src/__generated__/prisma';
 const prisma = new PrismaClient();
 
 async function main() {
-    const encryptedPassword = await bcrypt.hash('password', 10);
+    console.log('Dropping tables...')
+    await prisma.user.deleteMany();
+    await prisma.event.deleteMany();
+    await prisma.organization.deleteMany();
+
+    console.log('seeding DB...')
 
-    const testUser = await prisma.user.upsert({
+    const encryptedPassword = await bcrypt.hash('password', 10);
+    await prisma.user.upsert({
```

Comment:
@kadey001 IMO some of this pretest stuff should be utility functions instead. It would be slightly repetitive but it would also give finer grained control over the tests as a whole and each environment for each test.